odoo/addons/test_lint/tests/test_manifests.py

```diff
@@ -0,0 +1,16 @@
+# Part of Odoo. See LICENSE file for full copyright and licensing details.
+
+from odoo.modules import get_modules
+from odoo.modules.module import load_manifest, _DEFAULT_MANIFEST
+from odoo.tests import BaseCase
+
+class ManifestLinter(BaseCase):
+
+    def test_manifests_keys(self):
+        whitelist_keys = {'name', 'icon', 'addons_path', 'license', 'bootstrap', 'maintainer', 'url', 'web.assets_backend' }
+        known_keys = set(_DEFAULT_MANIFEST.keys())
```

Comment:
```suggestion
        known_keys = _DEFAULT_MANIFEST.keys()
```
`dict_keys` implement `Set` (does not extend `set`).

Reply:
Alternatively, or additionally, why not just merge these two sets immediately? They're getting subtracted from the manifest keys either way.